src/test/kotlin/no/nav/aap/api/søknad/SøknadTest.kt

```diff
@@ -178,7 +178,7 @@ class SøknadTest {
                                 UUID.randomUUID()))), listOf(AnnenStønad(INTRODUKSJONSSTØNAD))),
                 "Tillegg",
                 listOf(BarnOgInntekt(true)),
-                listOf(AnnetBarnOgInntekt(Barn(Navn("Et", "ekstra", "Barn"), now().minusYears(14)),FORELDER,false,Vedlegg(deler = listOf(UUID.randomUUID())))), Vedlegg(deler = listOf(UUID.randomUUID(),
+                listOf(AnnetBarnOgInntekt(Barn(Navn("Et", "ekstra", "Barn"), now().minusYears(14)),FORELDER,false,null,/*Vedlegg(deler = listOf(UUID.randomUUID()))*/)), Vedlegg(deler = listOf(UUID.randomUUID(),
```

Comment:
Er det meningen her å kommentere ut vedlegg for testen sin skyld? eller snakker vi quickfix? 😅

Reply:
Det spiller ingen rolle for testen, jeg prøvde å få inn to manglende vedlegg